### ISSUE TITLE
support hyperloglog ? 

### ISSUE THREAD

**Issue by rfyiamcool** [debasishg/scala-redis#146]
support hyperloglog ?

**Comment by debasishg** [comment]
https://github.com/debasishg/scala-redis/blob/master/src/main/scala/com/redis/HyperLogLogOperations.scala

**Comment by dskrvk** [comment]
Duplicate: https://github.com/debasishg/scala-redis/issues/125

**Comment by debasishg** [comment]
yes .. https://github.com/debasishg/scala-redis/blob/master/src/main/scala/com/redis/HyperLogLogOperations.scala
